packages/react/src/views/ChatHeader/ChatHeader.js

```diff
@@ -158,12 +160,15 @@ const ChatHeader = ({
       }
     };
 
+    
     const getChannelInfo = async () => {
       const res = await RCInstance.channelInfo();
+      const userRes = await RCInstance.getAuth();
+      const isAdmin = userRes.currentUser.me.roles.includes('admin');
```

Comment:
Even for admins, we want to display the 'This room is read-only' label. We should only enable the input box for admins in that case (which is disabled for all currently). Additionally, we can't just include admins; it depends on the workspace admin to allow people to message in read-only channels. If a role includes any permissions related to that, it should enable messaging

![image](https://github.com/RocketChat/EmbeddedChat/assets/78961432/7402e4ae-9710-4753-be46-04c0f01edc1a)

Reply:
Did you mean this label?
![image](https://github.com/RocketChat/EmbeddedChat/assets/125076549/67121581-4611-4cf3-8f01-d1fb53a4d5f0)

Reply:
Yes

Reply:
Is it hidden in Rocket.chat main application for admins too ? Once check and let me know !

Reply:
Also, if you will see a variable : const allRoles = [...channelLevelRoles, ...workspaceLevelRoles]; Just use it and compare with PostReadOnly roles